### DISCUSSION TITLE
OpenSearch compatibility with apiman

### DISCUSSION THREAD

**Discussion by shachindrasingh** [apiman/apiman#2340] [Enhancement proposals]
Hi,
is apiman 2.2.3 is compatibility with openSearch or any plan to make this compatibility in upcoming release ? please confirm.

**Comment by volkflo** [comment]
@shachindrasingh Apiman is compatible with OpenSearch/OpenDistro for Elasticsearch and Elasticsearch itself.
With the upcoming 3.x release, ES cannot be used as a backend storage. It can still be used for metrics, etc.
